src/index.ts

```diff
@@ -8,45 +51,25 @@
  * ```
  * @param idNumber norwegian social security number
  */
-export const NorwegianId = (idNumber: string) => {
+export const NorwegianId = (idNumber: string): NorwegianIdObject => {
```

Comment:
OK with me, but I couldn't find that the previous type was resolved to `any` (ref your comment), I got that it resolved to an object:

![image](https://user-images.githubusercontent.com/2505178/151715674-4db38291-c906-4941-8567-bc44963e42f3.png)

Reply:
It resolves to an object. But if you try calling the functions on the object the inferred type was `any` (for some reason) 🤷